files/zip.go

```diff
@@ -9,16 +9,27 @@ import (
 	"path/filepath"
 	"time"
 
+	"github.com/amp-labs/cli/logger"
 	"github.com/amp-labs/cli/utils"
 )
 
 var now = time.Now() //nolint:gochecknoglobals
 
+const (
+	TmpDir     = ".amp" // This is used for zipping and uploading
```

Comment:
The optimal solution would be to just not use the working directory at all when creating temp files, since even if we try our best to clean up, sometimes the CLI might crash unexpectedly. 

I recommend using https://github.com/adrg/xdg (which we already use in the appdata folder) to create the zip files in a well-known location that's hidden away (maybe XDG_CACHE_HOME and XDG_DATA_DIRS).

Reply:
Alternatively, you could do everything in memory. See https://github.com/amp-labs/server/blob/main/scripts/endToEnd.go#L676-L714 for an example of how I do this.

I like this in terms of not creating cruft on the disk. But if we need the intermediate zip files for some reason I guess it's not the best option.

Reply:
I chose to do it in memory since that involved the least amount of changes + works better. Please review.